alf/test/main.py

```diff
@@ -16,6 +16,8 @@
 import torch
 import unittest
 
+import alf
+
```

Comment:
These two lines can be removed?

Reply:
Good catch.  It was from a git pick from an outdated branch..